omnibus/resources/agent/msi/source.wxs.erb

```diff
@@ -404,7 +404,7 @@
       <!-- only do the uninstall (which removes the user state and file perms) 
            on a full uninstall -->
       <Custom Action='SetUninstallProperty' Before='DoUninstall'> <![CDATA[REMOVE="ALL" AND UPGRADINGPRODUCTCODE=""]]> </Custom>
-      <Custom Action='DoUninstall' Before='InstallFinalize'>      <![CDATA[REMOVE="ALL" AND UPGRADINGPRODUCTCODE=""]]> </Custom>
+      <Custom Action='DoUninstall' Before='RemoveRegistryValues'>      <![CDATA[REMOVE="ALL" AND UPGRADINGPRODUCTCODE=""]]> </Custom>
```

Comment:
@derekwbrown is the change on this line related to your PR's purpose, or is this fixing something else? 🤔 Do we need to QA this change separately/specifically?